en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-the-integrated-api-console.md

```diff
@@ -1,12 +1,12 @@
 # Invoke an API using the Integrated API Console
 
-WSO2 API Manager (WSO2 API-M) has an integrated Swagger UI, which is part of the Swagger project.
+WSO2 API Manager (WSO2 API-M) has an integrated Swagger UI, which renders OpenAPI specification as interactive API documentation.
```

Comment:
Shall we reword this as 
The API Console allows you to visualize the API contract and interact with API's resources without being aware of the backend logic. The API contract is defined in Open API Specification(OAS), and it is rendered in API Console as interactive and consumer-focused API documentation.